auth-web/src/components/auth/account-settings/statement/StatementsSettings.vue

```diff
@@ -160,31 +178,33 @@
       </v-card>
     </v-dialog>
     <v-snackbar
+      v-model="showStatementNotification"
       bottom
       multi-line
       :timeout="6000"
       class="mb-6"
-      v-model="showStatementNotification"
     >
       Statement Settings updated
       <v-btn
         dark
         icon
         aria-label="Close Notification"
         title="Close Notification Message"
-        @click="showStatementNotification = false">
+        @click="showStatementNotification = false"
+      >
         <v-icon>mdi-close</v-icon>
       </v-btn>
     </v-snackbar>
   </div>
 </template>
 
 <script lang="ts">
-import { Component, Mixins, Prop, Vue, Watch } from 'vue-property-decorator'
+import { Component, Vue } from 'vue-property-decorator'
 import { Member, MembershipType, Organization } from '@/models/Organization'
 import { StatementListItem, StatementNotificationSettings, StatementRecipient, StatementSettings } from '@/models/statement'
 import { mapActions, mapState } from 'vuex'
 import CommonUtils from '@/util/common-util'
+/* eslint-disable-next-line @typescript-eslint/no-unused-vars */
```

Comment:
If moment isn't included for some of these components I remember seeing runtime errors.